<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>279</width>
    <height>384</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reiniciar</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Bip Bip</string>
       </property>
       <widget class="QLCDNumber" name="lcdNumber_bip">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_more_bip">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Bip Bip</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Digamo</string>
       </property>
       <widget class="QLCDNumber" name="lcdNumber_digamo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_more_digamo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Digamo</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_new_window">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>340</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nueva ventana</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
